app/models/support/requests/eu_exit_business_readiness_request.rb

```diff
@@ -122,6 +52,16 @@ def self.label
       def self.description
         "Request to add content, update content tags or remove content from the EU Exit business readiness finder."
       end
+
+    private
+
+      def raw_facet_group
+        RemoteFacetGroupService.new.find(EU_EXIT_BUSINESS_FINDER_CONTENT_ID)
```

Comment:
Not a blocker for this PR as this method is only every called once per support request but this could be memoized so that we don't inadvertently re-request the facet group. eg.

```ruby
@facet_group ||= RemoteFacetGroupService.new.find(EU_EXIT_BUSINESS_FINDER_CONTENT_ID)
```